authentik/events/enrich/geoip.py

```diff
@@ -23,7 +26,7 @@ class GeoIPDict(TypedDict):
     city: str
 
 
-class GeoIPReader:
+class GeoIPEnricher(EventEnricher):
```

Comment:
It seems there's a lot of duplicated stuff between the GeoIPEnricher and ASNEnricher. I get it doesn't make sense to but those in the baseenricher, but does it make sense to make a MMDBEnricher?